src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <label>
+      <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      name="form-field"
+      data-qa="big"
+      class="form"
+      method="post"
+      >
```

Comment:
```suggestion
      <form
        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
        name="form-field"
        data-qa="big"
        class="form"
        method="post"
      >
```
pay attention to the code style